kratos/python_scripts/print_info_in_file_process.py

```diff
@@ -0,0 +1,131 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return PrintInfoInFileProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class PrintInfoInFileProcess(KratosMultiphysics.OutputProcess):
+    """This process prints a text file with the required nodal or elemental information
+
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+
+    def __init__(self, Model, settings ):
+        """ The default constructor of the class
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+        super().__init__()
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "help"                     : "This process prints nodal/elemental information ina .txt file",
+            "mesh_id"                  : 0,
+            "model_part_name"          : "please_specify_model_part_name",
+            "variable_name"            : "SPECIFY_VARIABLE_NAME",
+            "variable_type"            : "nodal_historical",
+            "file_name"                : "info_file.txt",
+            "output_control_type"      : "step",
+            "integration_point_number" : 0,
+            "erase_previous_info"      : true,
+            "output_interval"          : 1,
+            "sum_results_from_multiple_entites" : false
+        }""")
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
+        self.variable_type = settings["variable_type"].GetString()
```

Comment:
I think that in here we should check the provided `self.variable_type` to be either `nodal_historical`, `nodal_non_historical` and `elemental`. If not we should throw an error asking the user to provide one of these. Note that as it is right now providing an empty string (or a random one) will end up printing the elemental values.

Reply:
check added and removed the default variable type to ensure that the user sets it

Reply:
ah and can you please rename this?
we use this name usually for e.g. `bool`, `double` etc